components/style/values/specified/list.rs

```diff
@@ -0,0 +1,64 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+//! `list` specified values.
+
+use cssparser::{Parser, Token, serialize_string};
+use parser::{Parse, ParserContext};
+use std::fmt;
+use style_traits::{ParseError, StyleParseErrorKind, ToCss};
+
+/// Specified and computed `quote` property
+#[derive(Clone, Debug, MallocSizeOf, PartialEq, ToComputedValue)]
+pub struct Quotes(pub Vec<(String, String)>);
```

Comment:
These strings and this Vec should probably be box, to save space:

```rust
pub struct Quotes(pub Box<[(Box<str>, Box<str>)]>);
```

Not sure if you feel like fixing it, but if not please file an issue :)